tests/app/test_jinja_filters.py

```diff
@@ -310,6 +310,24 @@ def test_format_household_member_name_possessive_trailing_s(self):
 
         self.assertEqual(format_value, 'John Does\u2019')
 
+    def test_concatenated_list(self):
```

Comment:
can you add a test with 1 item in the list?

Reply:
Yes, sorry my bad was supposed to do it as discussed